updated NORAL details (#137)
CivicDataLab/civicdatalab.github.io: content/work/education/noral/index.md

---
name: Democratising Action for Attainment
image: ./image.png
summary: Understanding attainment through the lens of poverty across the Northern Alliance region of Scotland.

> context: The circumstances in which a child grows up strongly influences their outcomes across Scotland. Impacts of child poverty on the attainment of a child which need to be holistically assessed to better understand how it affects them. There are structural inequalities in the resources available to these children and the support provided to best utilise them.

Reviewer: ```suggestion
context: The circumstances in which a child grows up strongly influence their outcomes across Scotland. The impact of child poverty on the attainment of a child needs to be holistically assessed to better understand how it affects them. There are structural inequalities in the resources available to these children and the support provided to best utilise them.
```

> aim: Education and children services have been working on this problems for years to understand the challenges faced by a child growing in the different local contexts by attempting to follow the journey of a child through the system. One way they are trying to achieve this is by interacting with various sources of data and platforms available to them that help understand the same.

Reviewer: ```suggestion
aim: Education and children services have been working on these problems for years to understand the challenges faced by a child growing in the different local contexts by attempting to follow the journey of a child through the system. One way they are trying to achieve this is by interacting with various sources of data and platforms available to them that help them understand the same.
```

> solution: All our research lies on the intersection of `data`, `tech`, `design` and `social science` to understand all aspects of the ecosystem and suggest solutions that strengthen the course of civic engagements. Through our research we interacted with the stakeholders to understand their role in the ecosystem, the tools their are engaging with, their current experiences and their aspirations for an ideal future. Through all these inputs were able to identify the preferences of various stakeholders and their requirements from a new platform.

Reviewer: ```suggestion
solution: All our research lies on the intersection of `data`, `tech`, `design`, and `social science` to understand all aspects of the ecosystem and suggest solutions that strengthen the course of civic engagements. Through our research, we interacted with the stakeholders to understand their role in the ecosystem, the tools they are engaging with, their current experiences, and their aspirations for an ideal future. Through all these inputs were able to identify the preferences of various stakeholders and their requirements from a new platform.
```
